wardrobe/timestamp/src/cleanup_tests.rs

```diff
@@ -1,102 +1,120 @@
 //! Unit tests for the Timestamp piece.
 //! This module tests the secondary flow of cleaning up old timestamps.
 
-use super::{
-    BestTimestamp, CleanUpTimestamp, NotedTimestamp, SimpleConstraintChecker, TimestampConfig,
-    TimestampError,
-};
+use super::*;
```

Comment:
I used the longer imports so that I _didn't_ get the `ConstraintChecker` trait in scope. By only having `SimpleConstraintChecker` in scope, you don't need to specify which trait you're trying to call in each test.

Reply:
It's better to do how it was done before, sorry for the mess. Should I fix it, or are you taking care of this?

Reply:
I'm not working on it. I also don't really care that much. Just left the comment for your understanding. Feel free to change it back if you want, but not a big deal for me.

Reply:
I think it should be considered a good practice, since it keeps the code both more coincise and more readable, and it should be kept as an example for posterity

Reply:
solved in 52b2f855ba30ce70a1ea2ca70ad6583b29bcc19e